pkg/machines/hostvmslist.jsx

```diff
@@ -465,8 +441,30 @@ class HostVmsList extends React.Component {
             allActions = allActions.concat(actions);
         }
 
+        const errors = [...ui.errors];
+        const isLibvirtActive = libvirtState.status === 'active';
+
+        if (!isLibvirtActive) {
```

Comment:
I would make the `NotificationArea` component responsible for producing this message, just pass state-info to props.